vita3k/gui/src/game_selector.cpp

```diff
@@ -63,11 +63,10 @@ bool refresh_game_list(GuiState &gui, HostState &host) {
     return true;
 }
 
-static constexpr auto MENUBAR_HEIGHT = 19;
-
 void draw_game_selector(GuiState &gui, HostState &host) {
     const ImVec2 display_size = ImGui::GetIO().DisplaySize;
     static std::string title_id;
+    static constexpr auto MENUBAR_HEIGHT = 19;
```

Comment:
```suggestion
    constexpr auto MENUBAR_HEIGHT = 19;
```

Reply:
is already static on master 🤔 i just move it inside function

Reply:
i know

Reply:
And i keep use static because it not need run few time, so value is read one time, more speed

Reply:
lol no
just do what i said

Reply:
I see, i have just wanted keep like master and like petmac have explain me, with using static on function if it never changed